client/components/domains/contact-details-form-fields/custom-form-fieldsets/utils.js

```diff
@@ -0,0 +1,24 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+export const STATE_LABEL = {
+	CA: translate( 'Province' ),
+};
+export const STATE_SELECT_TEXT = {
+	CA: translate( 'Select Province' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **53** times:
`translate( 'Province' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).